server/graphql-engine.cabal

```diff
@@ -215,6 +215,16 @@ library
                      , Hasura.GraphQL.Transport.WebSocket.Protocol
                      , Hasura.GraphQL.Transport.WebSocket.Server
                      , Hasura.GraphQL.Transport.WebSocket
+                     , Hasura.GraphQL.Schema.BoolExp
```

Comment:
The code from `Schema.hs` is more or less moved into these new modules (except in couple of cases, documented later).